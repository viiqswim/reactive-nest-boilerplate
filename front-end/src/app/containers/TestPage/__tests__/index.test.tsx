import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { TestPage } from '..';

const shallowRenderer = createRenderer();

describe('<TestPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<TestPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
