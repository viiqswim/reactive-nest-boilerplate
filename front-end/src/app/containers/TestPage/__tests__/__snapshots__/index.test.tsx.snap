// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Test Page
    </title>
    <meta
      content="A React Boilerplate application homepage"
      name="description"
    />
  </Helmet>
  <NavBar />
  <ForwardRef(PageWrapper)>
    <Masthead />
    <SignUp />
  </ForwardRef(PageWrapper)>
</React.Fragment>
`;
